backend/templates/templated_email/signup_invitation.email

```diff
@@ -0,0 +1,10 @@
+{% block subject %}PokeBattle - Signup to battle with {{invitee}}{% endblock %}
+
+{% block plain %}
+    Hi {{ invited }}, {{ invitee }} wants to play a pokemon battle with you. Sign up at https://pantoja-pokebattle.herokuapp.com/ to play pokebattle with friends!
+{% endblock %}
+
+{% block html %}
+    <p>Hi {{ invited }},</p>
+    <p>{{ invitee }} wants to play a pokemon battle with you. Sign up at <a href="https://pantoja-pokebattle.herokuapp.com/">Pokebattle</a> to play pokebattle with friends!</p>
```

Comment:
This link is pointing to the wrong instance of Pokébattle as the URL is hardcoded. This is not ideal because in a real situation, we usually have staging and production instances, and staging would always have URLs that pointed to production. Please make this dynamic, see here https://stackoverflow.com/questions/2345708/how-can-i-get-the-full-absolute-url-with-domain-in-django By getting the absolute URL you'll be able to have access to the domain without hardcoding anything.